src/constants.js

```diff
@@ -0,0 +1,5 @@
+// constants.js
+
+const apiKey = '2a878fc593044fdba0c5153f9a2a46e7';
+
+export default apiKey;
```

Comment:
it's better to export object because apikey is only one possible constant, or you can use non default export